### PR TITLE
SCSS Deprecation

### PR DESCRIPTION
**Pull request by lancetharper**
- SCSS deprecation file
- Add deprecation/migration info to tokens
- Formats, etc. to support deprecation output
- README updates with notes about adding docs and deprecation data to tokens

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,11 @@
 
     ** REI Internal teams should use the internal [iOS](https://git.rei.com/projects/CDR2/repos/rei-cedar-ios/) and [Android](https://git.rei.com/projects/CDR2/repos/rei-cedar-android/) packages.
 
+## Updating
+
+If you are consuming tokens in SCSS, there are deprecation warnings for variables, mixins, etc. that will appear in the console during your build. These can be silenced by adding a variable to your code called `cdr-warn` and setting it to false like this: `$cdr-warn: false;`
+
+All other formats should consult the changelog for a migration path.
```
Comment (KrisKnabel) [inline README.md]: I like this! 👍

File: README.md [hunk 2]
```diff
@@ -149,8 +153,87 @@ See [attribute referencing](https://amzn.github.io/style-dictionary/#/properties
...
+```
+
+### Deprecating tokens
+
+Deprecated tokens should be moved to a seprate file (or into the existing file) which corresponds to the release cycle in which they will be deprecated.
+
+For example, if tokens will be considered deprecated in the "Winter 2019" release they would be moved into a file called `deprecated-2019-winter.json5` in whichever directory they currently reside. Structure for naming the file is : `deprecated-<year>-<release>`
+
+Additionally, the contents will be wrapped inside an object with a key that corresponds to the release as well (so we can auto generate some deprecation warnings with the correct release). The key matches the naming of the file. See below for an example.
+
```
Comment (KrisKnabel) [inline README.md]: Do we have any guidelines on what to do to the value of the deprecated token?  Do we map it to new value? Keep it as the original value? Is it a case by case decision?  Mostly wondering if there is a convention to follow.

File: style-dictionary/build.js [hunk 3]
```diff
@@ -1,11 +1,11 @@
 const StyleDictionary = require('style-dictionary');
-const path = require('path');
+// const path = require('path');
 const getPlatformConfig = require('./configs');
 // const _ = require('lodash');
```
Comment (KrisKnabel) [inline style-dictionary/build.js]: Wanna keep the commented out dependencies?

File: README.md (hunk 2 above)
Comment (lancetharper) [reply README.md]: I believe the recommendation is to map it to a new value if possible. This allows for that and will inform the user of the new recommended token/mixin if provided.

**Pull request merged**